pro/src/components/IndividualOffer/StocksEventCreation/form/onSubmit.ts

```diff
@@ -56,14 +57,21 @@ export const onSubmit = async (
       notify.success(
         stocks_count > 1
           ? `${new Intl.NumberFormat('fr-FR').format(
-              stocks_count
-            )} nouvelles dates ont été ajoutées`
+            stocks_count
+          )} nouvelles dates ont été ajoutées`
           : `${stocks_count} nouvelle date a été ajoutée`
       )
-    } catch {
-      notify.error(
-        'Une erreur est survenue lors de l’enregistrement de vos stocks.'
-      )
+    } catch (error) {
+      if (
+        error instanceof ApiError &&
+        error.body['stocks'] instanceof Array
+      ) {
+        notify.error(error.body['stocks'][0])
+      } else {
+        notify.error(
+          'Une erreur est survenue lors de l’enregistrement de vos stocks.'
+        )
+      }
```

Comment:
Normalement, tu pourrais remplacer tout ça par ça, mais tu affiches les erreurs autres que celles des stocks.

```suggestion
      getHumanReadableApiError(error, 'Une erreur est survenue lors de l’enregistrement de vos stocks.')
```